SwiftLeeds/App/SwiftLeedsApp.swift

```diff
@@ -9,10 +9,6 @@ import SwiftUI
 
 @main
 struct SwiftLeedsApp: App {
-    init() {
```

Comment:
This does drop us to the default translucent bar, but I'd rather that than broken accent colors everywhere